plugin.php

```diff
@@ -84,7 +84,7 @@
  */
 include_once( dirname( __FILE__ ) . '/extras.php' );
 
-add_filter( 'register_post_type_args', '_add_extra_api_post_type_arguments', 10, 2 );
+add_filter( 'init', '_add_extra_api_post_type_arguments', 11 );
```

Comment:
I actually like this approach better anyway, since it means we're not running a bunch of times when we don't need to.